fenix/app/src/main/java/org/mozilla/fenix/addons/AddonsManagementFragment.kt

```diff
@@ -274,6 +274,6 @@ class AddonsManagementFragment : Fragment(R.layout.fragment_add_ons_management)
         // This is locale-less on purpose so that the content negotiation happens on the AMO side because the current
         // user language might not be supported by AMO and/or the language might not be exactly what AMO is expecting
         // (e.g. `en` instead of `en-US`).
-        private const val AMO_HOMEPAGE_FOR_ANDROID = "https://addons.mozilla.org/android/"
+        private const val AMO_HOMEPAGE_FOR_ANDROID = BuildConfig.AMO_BASE_URL + "/android/"
```

Comment:
I should have done that in the previous patch, sorry...

Reply:
No worries!
Hint: We also have strings interpolation in [Kotlin](https://kotlinlang.org/docs/java-to-kotlin-idioms-strings.html#concatenate-strings) too :) 

Not a suggestion to update the code, just making you aware in case you were not familiar :) 
This also could be  `"${BuildConfig.AMO_BASE_URL}/android/"`.